pipelines/components/tensorflow/kmeansclustering/work.py

```diff
@@ -0,0 +1,51 @@
+from tensorflow.contrib.factorization import KMeansClustering
+
+import argparse
+import inspect
+import os
+import yaml
+from utils import get_args
+
+# TODO: implement appropriate functions for each generated component.
+from model import make_estimator
+from input_fn import make_input_fn
+from train import train
+
+
+def main(args):
+    # turn args into a dict so we can pop the arguments.
+    args = vars(args)
+
+    # create estimator
+    estimator_args = get_args(args, KMeansClustering.__init__)
+    estimator = make_estimator(KMeansClustering, estimator_args)
+
+    # create input_fn
+    make_input_fn_args = get_args(args, make_input_fn)
+    input_fn = make_input_fn(**make_input_fn_args)
+
+    # train, pass the remaining args
+    train(estimator, input_fn, **args)
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--npy_uri', default='gs://sandboxdata/cmle-pipelines/X.npy', type=str)
```

Comment:
Adding some help to these arguments would be a good idea to explain what they are.